db/migrate/20240929170706_create_service.rb

```diff
@@ -0,0 +1,15 @@
+class CreateService < ActiveRecord::Migration[7.1]
+  def change
+    create_table :services do |t|
+      t.references :doctor_profile, foreign_key: true
+      t.references :establishment, foreign_key: true
+      t.string :name, null: false
+      t.money :amount, null: false
+      t.integer :day_of_week, null: false
```

Comment:
This field should handle mutiple days of week. So if you are using integer then it will be like this:
If doctor selects Mon-Fri this integer value will be `01234` each digit representing a day of week.

Based on this maybe we can keep this string only and still use `01234`. The advantage of this would be when we will read this column value and from that try to decode what all days this service is open it will be easy to decode with string compared to integer.

Let me know your thoughts.

Ideally it should have been a separate table but lets not in depth right now.

Reply:
Fixed it by changing type to string from integer